app/scripts/AssistiveRehab-TUG_SIM.xml.template

```diff
@@ -14,7 +14,7 @@
 
     <module>
         <name>gazebo</name>
-        <workdir>$ENV{ROBOT_CODE}/assistive-rehab/app/conf</workdir>
+        <workdir>$ENV{ROBOT_CODE}/assistive-rehab/app/gazebo/tug</workdir>
```

Comment:
Uhm... I would avoid using env variables here as well as the `workdir`, which is quite an antipattern.

If we know that the SDF file is located in a path pointed by `GAZEBO_MODELS_PATH` or similar, we don't need to specify the `workdir` option: is that correct?

Reply:
Calling for @traversaro who might give hints on how starting up Gazebo from an application without resorting to env variables.

Reply:
`GAZEBO_MODEL_PATH` is where `gazebo` looks for models.
`tug-scenario` is a world application, I just found out that we can specify an additional env variable `GAZEBO_RESOURCE_PATH`, where gazebo looks for worlds. I'll update the `xml` and point this in the tutorial.

Reply:
I think @vvasco  was using `workdir` to correctly load the `tug-scenario.sdf` Gazebo world passed via the commandline. As you can read in http://gazebosim.org/tutorials?tut=components&cat=get_started, you can append to the `GAZEBO_RESOURCE_PATH` enviroment variable to the directory that contains the `.world` files that you  want to load, to avoid the need to use `workdir`. 
Note that unfortunately due to a Gazebo bug if you append a value to the `GAZEBO_RESOURCE_PATH`, you need to make sure to have called the `setup.sh` script of Gazebo before hand, see https://bitbucket.org/osrf/gazebo/issues/1607/gazebo-does-not-start-if-the .

Reply:
thanks @traversaro!